### PR TITLE
logs the username of the request sent to the api.

### PR DESCRIPTION
**Pull request by tdowds**
Logs the username for each request sent to the api

### CODE REVIEW

File: huxley/core/middlewares.py [hunk 1]
```diff
@@ -20,7 +20,8 @@ def process_response(self, request, response):
             log = json.dumps({
                   'message': "Logging response from the api.",
                   'uri': request.path,
-                  'status_code': response.status_code})
+                  'status_code': response.status_code,
+                  'username': request.user.username})
```
Comment (kmeht) [inline huxley/core/middlewares.py]: Will this throw if the user isn't logged-in? i.e. an anonymous user?

File: huxley/logging/models.py [hunk 2]
```diff
@@ -11,6 +11,7 @@ class LogEntry(models.Model):
 
     uri = models.CharField(max_length=200)
     status_code = models.PositiveSmallIntegerField(default=0)
+    username = models.CharField(max_length=200)
```
Comment (kmeht) [inline huxley/logging/models.py]: We need a migration for this too.

File: huxley/core/middlewares.py (hunk 1 above)
Comment (tdowds) [inline huxley/core/middlewares.py]: It returns an empty string

**Comment by nathanielparke** [comment]
Merged via 72f76ae12f4b8836931f6ddbf3a3a0eb75114862

**Pull request closed without merging**